content/applications/sales/crm/acquire_leads/generate_from_email.rst

```diff
@@ -1,20 +0,0 @@
-========================================
-Generate leads/opportunities from emails
-========================================
-
-Automating the lead/opportunity generation will considerably improve
-your efficiency. By default, any email sent to
-*sales@database\_domain.ext* will create an opportunity in the
-pipeline of the default sales channel.
-
-Configure email aliases
-=======================
-
-Each sales teams can have its own email alias, to generate
-leads/opportunities automatically assigned to it. It is useful if you
-manage several sales teams with specific business processes. You will
-find the configuration of sales teams under
-:menuselection:`Configuration --> Sales Teams`.
-
-.. image:: media/generate_from_email01.png
```

Comment:
Shouldn't you remove images referenced by `generate_from_email` too?

Reply:
The `generate_from_email.rst` file was deleted and folded into the now renamed `generate_leads.rst`.